src/component/contents/DraftEditorBlock.react.js

```diff
@@ -103,22 +119,18 @@ class DraftEditorBlock extends React.Component {
       var nodeBottom = nodePosition.y + nodePosition.height;
       var viewportHeight = getViewportDimensions().height;
       scrollDelta = nodeBottom - viewportHeight;
-      if (scrollDelta > 0) {
```

Comment:
The initial thought was that the this was preventing scrolling up if you selected something near the top of the document, scrolled down, then started typing. I'm realizing now that removing this line also causes scrolling when we don't want it - I'll rethink a bit